### PR TITLE
Update kibana app package.json path

### LINKED ISSUE #2247: Update packages builder for wazuh app (plugins)
## Description

From 4.6.0, we have adopted a new folder structure in the `wazuh-kibana-app` repo. The source code of the plugin is now inside the `plugins/main `folder, `package.json` file included.

These changes have collateral effects [packages builder scripts](https://github.com/wazuh/wazuh-packages/tree/4.6.0/wazuhapp), so they need to be updated.

Related issue: https://github.com/wazuh/wazuh-kibana-app/issues/5566

### PR DESCRIPTION
**Pull request by Rebits**
|Related issue|
|---|
|closes #2247 |

## Description

This PR updates the `package.json` path for the Wazuh Kibana app.

### CODE REVIEW

**Comment by Rebits** [comment]
### 04/07/2023
- Building Kibana Container locally

**Review by fcaffieri — DISMISSED** [review]

File: wazuhapp/kibana/Docker/build.sh [hunk 1]
```diff
@@ -103,10 +103,12 @@ install_dependencies () {
 
 
 download_wazuh_app_sources() {
-    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 ${kibana_dir}/plugins/wazuh ; then
+    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 /tmp/wazuh-app ; then
         echo "Error downloading the source code from wazuh-kibana-app GitHub repository."
         exit 1
     fi
+
+    cp -r /tmp/wazuh-app/plugins/main ${kibana_dir}/plugins/wazuh
```
Comment (jnasselle) [inline wazuhapp/kibana/Docker/build.sh:111]: Nitpick: `${kibana_dir}/plugins/wazuh` could be replaced with `${source_dir}`

File: wazuhapp/opensearch-dashboards/Docker/build.sh [hunk 2]
```diff
@@ -97,12 +97,13 @@ install_dependencies () {
...
 download_wazuh_app_sources() {
-    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 ${plugin_platform_dir}/plugins/wazuh ; then
-        echo "Error downloading the source code from wazuh-kibana-app GitHub repository."
+    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 /tmp/wazuh-app ; then
+        echo "Error downloading the source code from wazuh-dashboard-app GitHub repository."
         exit 1
     fi
-}
 
+    cp -r /tmp/wazuh-app/plugins/main ${plugin_platform_dir}/plugins/wazuh
```
Comment (jnasselle) [inline wazuhapp/opensearch-dashboards/Docker/build.sh:105]: Nitpick: `${plugin_platform_dir}/plugins/wazuh` could be replaced with `${source_dir}`

File: wazuhapp/opensearch-dashboards/Docker/build.sh [hunk 3]
```diff
@@ -97,12 +97,13 @@ install_dependencies () {
 
 
 download_wazuh_app_sources() {
-    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 ${plugin_platform_dir}/plugins/wazuh ; then
-        echo "Error downloading the source code from wazuh-kibana-app GitHub repository."
+    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 /tmp/wazuh-app ; then
```
Comment (jnasselle) [inline wazuhapp/opensearch-dashboards/Docker/build.sh:100]: Nitpick: how about a new variable like `git_clone_tmp_dir`

File: wazuhapp/kibana/Docker/build.sh [hunk 4]
```diff
@@ -103,10 +103,12 @@ install_dependencies () {
 
 
 download_wazuh_app_sources() {
-    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 ${kibana_dir}/plugins/wazuh ; then
+    if ! git clone $wazuh_app_clone_repo_url --branch ${wazuh_branch} --depth=1 /tmp/wazuh-app ; then
```
Comment (jnasselle) [inline wazuhapp/kibana/Docker/build.sh:106]: Nitpick: how about a new variable like `git_clone_tmp_dir`

**Review by jnasselle — CHANGES_REQUESTED** [review]
GJ! Just a few reading improvements

File: wazuhapp/kibana/Docker/build.sh (hunk 1 above)
Comment (Rebits) [reply wazuhapp/kibana/Docker/build.sh]: Done e197d57

File: wazuhapp/kibana/Docker/build.sh (hunk 4 above)
Comment (Rebits) [reply wazuhapp/kibana/Docker/build.sh]: Done e197d57

File: wazuhapp/opensearch-dashboards/Docker/build.sh (hunk 2 above)
Comment (Rebits) [reply wazuhapp/opensearch-dashboards/Docker/build.sh]: Done 8680554

File: wazuhapp/opensearch-dashboards/Docker/build.sh (hunk 3 above)
Comment (Rebits) [reply wazuhapp/opensearch-dashboards/Docker/build.sh]: Done 8680554

**Review by jnasselle — APPROVED** [review]

**Review by davidjiglesias — APPROVED** [review]

**Pull request merged**